src/mvtseries.jl

```diff
@@ -649,6 +651,24 @@ Base.view(x::MVTSeries, ::Colon, J::_SymbolOneOrCollection) = view(x, axes(x, 1)
 Base.view(x::MVTSeries, I::_MITOneOrRange, ::Colon=Colon()) = view(x, I, axes(x, 2))
 Base.view(x::MVTSeries, ::Colon, ::Colon) = view(x, axes(x, 1), axes(x, 2))
 function Base.view(x::MVTSeries, I::_MITOneOrRange, J::_SymbolOneOrCollection)
+    @boundscheck checkbounds(x, I)
+    @boundscheck checkbounds(x, J)
+    start, stop = _ind_range_check(x, I)
+    i1 = start:stop
+    i2 = _colind(x, J)
+    if I isa MIT
+        return view(_vals(x), first(i1), i2)
+    elseif J isa Symbol
+        return TSeries(first(I), view(_vals(x), i1, i2))
+    else
+        return MVTSeries(first(I), axes(x, 2)[i2], view(_vals(x), i1, i2))
```

Comment:
Yes, exactly. 

Both `view` and `MVTSeries` are "wrappers", in the sense that they don't allocate memory for an array, rather they hold another array and change how the user can access the elements of that array. 

So basically our `view` methods for `MVTSeries` here boil down to `view(MVTSeries(x)) = MVTSeries(view(x))`, if that makes sense, otherwise `view(MVTSeries(x)) = view(x)`. Similarly for `TSeries`